client/app/main/about/about.scss

```diff
@@ -1,4 +1,6 @@
 .about-view {
+  overflow-y: auto;
+  @include cfa-scrollbar(5px);
   .developers,
```

Comment:
so here you're using `@include` to import a set of styles under the name `cfa-scrollbar`?